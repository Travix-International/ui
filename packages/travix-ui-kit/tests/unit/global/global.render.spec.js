import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import React from 'react';
import Global from '../../../components/global/global';

describe('Global: render', () => {
  it('should return base active modal with close button', () => {
    const component = shallow(
      <Global>
        Global Content
      </Global>
    );
    expect(shallowToJson(component)).toMatchSnapshot();
  });

  it('should return base active modal with close button', () => {
    const component = shallow(
      <Global noscroll={false}>
        Global Content
      </Global>
    );
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
