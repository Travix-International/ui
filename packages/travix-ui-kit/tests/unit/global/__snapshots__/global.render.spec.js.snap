// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Global: render should return base active modal with close button 1`] = `null`;

exports[`Global: render should return base active modal with close button 2`] = `null`;
